packages/e2e-tests/helpers/commands.ts

```diff
@@ -15,4 +15,5 @@ export async function quickOpen(page: Page, url: string): Promise<void> {
   await debugPrint(page, `Opening file "${chalk.bold(url)}"`, "quickOpen");
   await page.keyboard.press("Enter");
   await page.waitForSelector(`[data-test-name="Source-${url}"]`);
+  await page.waitForSelector(`[data-test-name="Source"]`);
```

Comment:
Running locally I was seeing an exception in code using `getCurrentSource` because it wasn't waiting for this element to exist, but idk if this is the best fix